mavlink-bindgen/src/parser.rs

```diff
@@ -710,6 +714,108 @@ impl MavEnumEntry {
             .map(|d| d.emit_tokens())
             .unwrap_or_default()
     }
+
+    #[inline(always)]
+    fn emit_params(&self) -> TokenStream {
+        if let Some(params) = &self.params {
+            let any_value_range = params.iter().any(|p| {
+                p.min_value.is_some()
+                    || p.max_value.is_some()
+                    || p.increment.is_some()
+                    || p.enum_used.is_some()
+                    || (p.reserved && p.default.is_some())
+            });
+            let any_units = params.iter().any(|p| p.units.is_some());
+            let lines = params
+                .iter()
+                .map(|param| param.emit_doc_row(any_value_range, any_units));
+            let mut table_header = "| Parameter | Description |".to_string();
+            let mut table_hl = "| --------- | ----------- |".to_string();
+            if any_value_range {
+                table_header += " Values |";
+                table_hl += " ------ |";
+            }
+            if any_units {
+                table_header += " Units |";
+                table_hl += " ----- |";
+            }
+            quote! {
+                #[doc = ""]
+                #[doc = "# Parameters"]
+                #[doc = ""]
+                #[doc = #table_header]
+                #[doc = #table_hl]
+                #(#lines)*
+            }
+        } else {
+            quote!()
+        }
```

Comment:
You could do something like:

```
let Some(params) = &self.params else {
    return quote!();
}
```

and write the rest of the code with better readibility.